org.goldendict.desktop.yml

```diff
@@ -0,0 +1,146 @@
+rename-desktop-file: goldendict.desktop
+modules:
+  - name: opencc
+    buildsystem: cmake-ninja
+    build-options:
+      env:
+        CC: 'ccache gcc'
+        CXX: 'ccache g++'
+    sources:
+      - type: git
+        url: https://github.com/BYVoid/OpenCC.git
+        tag: ver.1.1.1
+        x-checker-data:
+          type: anitya
+          project-id: 7230
+          tag-template: ver.$version
+
+  - name: lzo2
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+    cleanup:
+      - /include
+      - /share/doc
+      - '*.la'
+      - /lib/*.so
+    sources:
+      - type: archive
+        url: 'https://www.oberhumer.com/opensource/lzo/download/lzo-2.10.tar.gz'
+        sha256: c0f892943208266f9b6543b3ae308fab6284c5c90e627931446fb49b4221a072
+        x-checker-data:
+          type: anitya
+          project-id: 1868
+          url-template: http://www.oberhumer.com/opensource/lzo/download/lzo-$version.tar.gz
+
+  - name: eb
+    config-opts:
+      - '--enable-shared'
+      - '--disable-static'
+    cleanup:
+      - /bin
+      - /share/doc
+    sources:
+      - type: git
+        url: https://salsa.debian.org/debian/eb.git
+        tag: debian/4.4.3-13
+        x-checker-data:
+          type: anitya
+          project-id: 155542
+          tag-template: debian/$version
+      # Update config.guess & config.sub to allow builds for aarch64 and others
+      # https://salsa.debian.org/debian/eb/-/merge_requests/3
+      - type: file
+        url: https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=6faca61810d335c7837f320733fe8e15a1431fc2
+        dest-filename: config.guess
+        sha256: 7d1e3c79b86de601c3a0457855ab854dffd15163f53c91edac54a7be2e9c931b
+      - type: file
+        url: https://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=6faca61810d335c7837f320733fe8e15a1431fc2
+        dest-filename: config.sub
+        sha256: b1016a1eb11cdf0903bbc533e42e8ac2cc4d54a80b0545f4f0af5a85198277e2
+
+  - name: zstd
+    buildsystem: meson
+    subdir: build/meson
+    config-opts:
+      - -Dbin_programs=false
+      - -Dbin_contrib=false
+    sources:
+      - type: git
+        url: https://github.com/facebook/zstd.git
+        tag: v1.4.8
+        x-checker-data:
+          type: anitya
+          project-id: 12083
+          tag-template: v$version
+
+  - name: libao
+    cleanup:
+      - /include
+      - '*.la'
+      - /lib/pkgconfig
+      - /lib/ckport
+      - /share
+    sources:
+      - type: archive
+        url: 'https://downloads.xiph.org/releases/ao/libao-1.2.0.tar.gz'
+        sha256: 03ad231ad1f9d64b52474392d63c31197b0bc7bd416e58b1c10a329a5ed89caf
+        x-checker-data:
+          type: anitya
+          project-id: 7629
+          url-template: 'https://downloads.xiph.org/releases/ao/libao-$version.tar.gz'
+
+  - name: goldendict
+    buildsystem: qmake
+    build-options:
+      env:
+        CC: 'ccache gcc'
+        CXX: 'ccache g++'
```

Comment:
What's the point here?

Reply:
To build faster locally? Or that's not used since it's qmake?

Reply:
And yes it doesn't work. Removed.